src/interaction/actionmanager_lua.inl

```diff
@@ -88,6 +88,16 @@ struct [[codegen::Dictionary(Action)]] Action {
     // not provided, the default value is /
     std::optional<std::string> guiPath;
 
+    /// This parameter, if specified, will be used as a hint to any potential user
+    /// interface as a desired background color. This can be used, for example, to
+    /// visually group similar Actions together
+    std::optional<glm::vec4> color [[codegen::color()]];
```

Comment:
The documentation for actions are not on the docs page afaik, so maybe we need a docs page for the actions? I don't think these comments end up in the docs page?

Or, we could make the comments to the registerAction lua function more explicit. Either way it would be nice to have the struct for Action on the docs page somehow

Reply:
Yea, it's a bigger redesign that is necessary that would go together with changing the `ghoul::Dictionary` in Lua functions to more explicit types as well.  I made an issue to track it #3552